modules/apps/27-interchain-accounts/controller/ibc_middleware_test.go

```diff
@@ -523,8 +524,8 @@ func (suite *InterchainAccountsTestSuite) TestOnRecvPacket() {
 				)
 
 				ctx := suite.chainA.GetContext()
-				ack := cbs.OnRecvPacket(ctx, path.EndpointA.GetChannel().Version, packet, nil)
-				suite.Require().Equal(tc.expPass, ack.Success())
+				res := cbs.OnRecvPacket(ctx, path.EndpointA.GetChannel().Version, packet, nil)
+				suite.Require().Equal(tc.expPass, res.Status == exported.Success)
```

Comment:
mega nit feel free to ignore, might be nicer to have a `suite.Require().Equal(tc.expStatus, res.Status, "expected x but got y")`

Reply:
updated!